test/integration/cli/api-description-cli-test.coffee

```diff
@@ -33,221 +36,220 @@ describe 'CLI - API Description Document', ->
     app.post '/machine-types', (req, res) ->
       res.send [{name: 'bulldozer'}]
 
-  beforeEach (done) ->
-    startServer configureServer, PORT, (err, serverInfo) ->
-      server = serverInfo
+    server = app.listen (err, info) ->
+      serverRuntimeInfo = info
       done(err)
 
   afterEach (done) ->
-    server.process.close(done)
+    server.close(done)
 
 
   describe 'When loaded from file', ->
 
     describe 'When loaded by glob pattern', ->
-      dreddCommand = undefined
-      args = ['./test/fixtures/single-g*t.apib', "http://127.0.0.1:#{PORT}"]
+      dreddCommandInfo = undefined
```

Comment:
Applies also for

- `api-description-cli-test.coffee`
- `api-blueprint-cli-test.coffee`
- `swagger-cli-test.coffee`